src/components/tables/ProjectTable.jsx

```diff
@@ -20,23 +21,38 @@ class ProjectTable extends React.Component {
     return ['Project', ...summaryFields, ''];
   };
 
-  getData = projectList => projectList.map((proj, i) => [
+  getData = projectList => projectList.map((proj, i) => {
+    var buttonText = "Submit Data"
+    if (useArboristUI) {
+      buttonText = this.userHasCreateOnProject(proj.name) ? "Submit/Browse Data" : "Browse Data";
+    }
+    return [
     proj.name,
     ...proj.counts,
     <Button
```

Comment:
just some picky things: could consider add CSS to these buttons so they will have same width